plugins/modules/vault_token_create.py

```diff
@@ -0,0 +1,213 @@
+  description:
+    - Creates a token in HashiCorp Vault, returning the response, including the token.
+  seealso:
+    - ref: community.hashi_vault.vault_token_create lookup <ansible_collections.community.hashi_vault.vault_token_create_lookup>
+      description: The official documentation for the C(community.hashi_vault.vault_token_create) lookup plugin.
+    - module: community.hashi_vault.vault_login
+    - ref: community.hashi_vault.vault_login lookup <ansible_collections.community.hashi_vault.vault_login_lookup>
+      description: The official documentation for the C(community.hashi_vault.vault_login) lookup plugin.
+    - ref: community.hashi_vault.vault_login_token filter <ansible_collections.community.hashi_vault.docsite.filter_guide.vault_login_token>
+      description: The official documentation for the C(community.hashi_vault.vault_login_token) filter plugin.
+  extends_documentation_fragment:
+    - community.hashi_vault.connection
+    - community.hashi_vault.auth
+    - community.hashi_vault.token_create
+  notes:
+    - Token creation is a write operation (creating a token persisted to storage), so this module always reports C(changed=True).
+    - "For the purposes of Ansible playbooks however,
+      it may be more useful to set C(changed_when=false) if you're doing idempotency checks against the target system."
+    - "In check mode, this module will not create a token, and will instead return a basic structure with an empty token.
+      However this may not be useful if the token is required for follow on tasks.
+      It may be better to use this module with C(check_mode=no) in order to have a valid token that can be used."
+  options: {}
+"""
+
+EXAMPLES = """
+- name: Login via userpass and create a child token
+  community.hashi_vault.vault_token_create:
+    url: https://vault:8201
+    auth_method: userpass
+    username: user
+    password: '{{ passwd }}'
+  register: token_data
+
+- name: Retrieve an approle role ID using the child token (token via filter)
+  community.hashi_vault.vault_read:
+    url: https://vault:8201
+    auth_method: token
+    token: '{{ token_data | community.hashi_vault.vault_login_token }}'
+    path: auth/approle/role/role-name/role-id
+  register: approle_id
+
+- name: Retrieve an approle role ID using the child token (token via direct dict access)
+  community.hashi_vault.vault_read:
+    url: https://vault:8201
+    auth_method: token
+    token: '{{ token_data.login.auth.client_token }}'
+    path: auth/approle/role/role-name/role-id
+  register: approle_id
+
+# implicitly uses token auth with a token from the environment
+- name: Create an orphaned token with a short TTL
+  community.hashi_vault.vault_token_create:
+    url: https://vault:8201
+    orphan: true
+    ttl: 60s
+  register: token_data
+
+- name: Display the full response
+  ansible.builtin.debug:
+    var: token_data.login
+"""
+
+RETURN = """
+login:
```

Comment:
```suggestion
login:
```
should we also add `sample:` ?

Reply:
re: the samples, I could either add a minimal sample that covers only the things called out in the return, or an example of a full(ish) structure that's likely to be returned, which is much bigger (see the `token_create_response.json` fixture in this PR). What do you think?

Reply:
do as you consider right, it's just a reminder:) maybe the minimal variant would be good